app/views/issues/_table.html.erb

```diff
@@ -61,7 +61,7 @@
               %>
             <% end %>
             <td class="column-actions">
-              <%= link_to edit_project_issue_path(current_project, issue) do %>
+              <%= link_to local_assigns[:qa] ? edit_project_qa_issue_path(current_project, issue) : edit_project_issue_path(current_project, issue) do %>
```

Comment:
It seems to me this will break the HTML fragment cache /cc @aapomm 

It'd be better if the relevant JS code updates this value, wouldn't it? Perhaps adding both links to the view / cache, and then having JS choose which one is loaded?

Reply:
replied in BC